scripts/transform.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Runs in CircleCI
+# Replaces all the http://localhost links of api-data to match the corresponding deployment location.
+
+if [ "${CIRCLE_BRANCH}" = 'master' ]; then # https://stackoverflow.com/a/2013589/3482533
+    ~/.local/bin/ditto transform --base-url='https://pokeapi.co' --src-dir=data --dest-dir=_gen
+elif [ "${CIRCLE_BRANCH}" = 'staging' ]; then
+    ~/.local/bin/ditto transform --base-url='https://pokeapi-test-b6137.firebaseapp.com' --src-dir=data --dest-dir=_gen
```

Comment:
Assuming Bash 4 is available on the host, I'd organize this with something like this (untested):

also rename the file to .bash instead of .sh if you do this to make it obvious it uses bash syntax

```bash
#!/usr/bin/env bash

declare -A BRANCH_TO_URL=(
    ["master"]="https://pokeapi.co"
    ["staging"]="https://pokeapi-test-b6137.firebaseapp.com"
)

BASE_URL="${BRANCH_TO_URL[$CIRCLE_BRANCH]}"
~/.local/bin/ditto transform --base-url="$BASE_URL" --src-dir=data --dest-dir=_gen
```